ThunderCats/UIAlertController+TCAdditions.h

```diff
@@ -27,12 +27,57 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "TCNullabilityMacros.h"
 
 @interface UIAlertController (TCAdditions)
 
+typedef void (^ TCCancelAction)(UIAlertAction * __tc_nonnull);
+
 /**
  *  This is a convenient method that returns an alert controller with the title, "Not Implemented", and message, "This feature is not yet implemented."
  */
-+ (instancetype)tc_notImplementedAlert;
++ (instancetype __tc_nonnull)tc_notImplementedAlert;
+
+/**
+ *  Creates a UIAlertController standard (popup) alert with a cancel button
+ *
+ *  @param title             The title of the alert
+ *  @param message           The message displayed on the alert
+ *  @param cancelButtonTitle The text displayed on the cancel button
+ *  @param cancelAction      A completion block which will be executed when the user taps cancel which has one UIAlertAction as a parameter
```

Comment:
Because this is a nullable argument, it might be worthwhile to describe what happens if `nil` is passed.